src/DevBetterWeb.Core/Entities/Book.cs

```diff
@@ -10,11 +10,13 @@ public class Book : BaseEntity, IAggregateRoot
 {
   public string? Title { get; set; }
   public string? Author { get; set; }
-  public string? Details { get; set; }
+	public int? MemberWhoUploadId { get; set; }
+	public string? Details { get; set; }
   public string? PurchaseUrl { get; set; }
   public int? BookCategoryId { get; set; }
   public int Rank { get; private set; }
 	public List<Member>? MembersWhoHaveRead { get; set; } = new List<Member>();
+	public Member? MemberWhoUpload { get; set; }
```

Comment:
You shouldn't need both an Id and a Member property. If you have Member you can always get MemberWhoUpload.Id, right?

Reply:
I made this for config and the EF migration but I think it is will create it automatically so yes I will remove it from here (id).